website/content/docs/k8s/observability/metrics.mdx

```diff
@@ -0,0 +1,115 @@
+---
+layout: docs
+page_title: Metrics
+sidebar_title: Metrics
+description: Metrics for Consul on Kubernetes
+---
+
+# Metrics
+
+Consul on Kubernetes integrates with Prometheus and Grafana to provide metrics for Consul Service Mesh, including: service metrics, sidecar proxy metrics, Consul agent metrics, and gateway metrics. Service metrics can also be seen in the Consul UI Topology Visualization view. This section documents how to enable each of these.
+
+**Note:** Metrics will be supported in Consul-helm `0.31.0` and Consul-k8s `0.25.0`. However, enabling the [metrics merging feature](#connect-service-and-sidecar-metrics-with-metrics-merging) with Helm value (`defaultEnableMerging`) or annotation (`consul.hashicorp.com/enable-metrics-merging`) can only be used with Consul `1.10.0-alpha1` and above. The other metrics configuration can still be used before Consul `1.10.0-alpha1`.
+
+## Connect Service and Sidecar Metrics with Metrics Merging
+
+Prometheus annotations are used to scrape metrics. Prometheus annotations only support scraping from one endpoint on a Pod, so Consul on Kubernetes supports metrics merging so the service metrics and sidecar proxy metrics are merged into one endpoint. If there are no service metrics, it also supports just scraping the sidecar proxy metrics.
+
+The diagram below illustrates how the metrics integration works when merging is enabled:
+
+[![Metrics Merging Architecture](/img/metrics-arch.png)](/img/metrics-arch.png)
+
+Connect service metrics can be configured with the Helm values nested under [`connectInject.metrics`](/docs/k8s/helm#v-connectinject).
+
+Metrics and metrics merging can be enabled by default for all connect-injectedPods with the following Helm values:
+```yaml
+connectInject:
+  metrics:
+    defaultEnabled: true
+    defaultEnableMerging: true
+```
+They can also be overridden on a per-Pod basis using the annotations `consul.hashicorp.com/enable-metrics` and `consul.hashicorp.com/enable-metrics-merging`.
+
```

Comment:
```suggestion
~> In most cases, the default settings will be sufficient. If you are encountering issues with colliding ports or service metrics not being merged, you may need to change the defaults.
```

Maybe something like this to tell most people they don't need to grok all of this? Otherwise it's kinda intimidating how much config there is.